app/containers/Landing/Landing.js

```diff
@@ -71,7 +83,8 @@ const Landing = function() {
 					The Underlay's purpose is to make knowledge accessible, connectable, and
 					inspectable in service of the public good. Like the web, it is an emergent
 					system and concept. It is built on a set of open source protocols and tools for
-					structuring, storing, and aggregating distributed graph data.
+					structuring, storing, and aggregating distributed graph data.  
+					<a href="https://notes.knowledgefutures.org/pub/underlay-short-intro/release/1"><i>Read more</i></a>
```

Comment:
I don't love the style this link produces on the header copy of the site.
<img width="800" alt="Screen Shot 2020-12-02 at 3 55 26 PM" src="https://user-images.githubusercontent.com/1000455/100930443-03bdfa00-34e1-11eb-95cc-dd6094a22b5f.png">

I'm also not totally sure that the RFC it links to is the best introduction. It references lots of outdated things (e.g. MIT's Knowledge Futures Group). My preference would be to add the two RFCs, but remove this edit from the PR.

Reply:
Yes I see -- better the same gray and proper whitespace. 
New v. pushed.

Perhaps we can post a modernized v of it to underlay.org/intro, as a non-technical introduction that can be linked from the word "concept" in this paragraph.